urlextract/urlextract_core.py

```diff
@@ -372,7 +373,7 @@ def _complete_url(self, text, tld_pos, tld, check_dns=False):
         :param str text: text where we want to find URL
         :param int tld_pos: position of TLD
         :param str tld: matched TLD which should be in text
-        :param bool check_dns: filter results to valid domains
+        :param bool check_dns: DEPRECATED filter results to valid domains
```

Comment:
We could live there and make it False when calling from `find_urls`. I'm just a bit worried because users should avoid use private methods and given this one would arguably offer something useful to the user, maybe it can became a smell... What about leaving the check_dns but make this a public method?

Reply:
I am sorry - it was probably too late at night when I wrote that.
I somehow thought that this is for `gen_urls()` however I still thing we should keep it without and not setting it as deprecated.